client/src/components/banner.jsx

```diff
@@ -80,6 +137,22 @@ class Banner extends React.Component {
       this.setState({hash: window.location.hash, buttonText })
     });
   }
+  sportHandle() {
+    let showSport = this.state.showSport === 1 ? 0 : 1;
+    this.setState({ showSport });
+  }
+  shopHandle() {
+    let showShop = this.state.showShop === 1 ? 0 : 1;
+    this.setState({ showShop })
+  }
+  storiesHandle() {
+    let showStories = this.state.showStories === 1 ? 0 : 1;
+    this.setState({ showStories })
+  }
+  activismHandle() {
```

Comment:
there is probably a way to make these functions DRYer by having one "handle" function that takes in the target as a parameter, but also dont change it if this is working for you.